### DISCUSSION TITLE
ボタンの修正案

### DISCUSSION THREAD

**Discussion by ishida-daiki** [ishida-daiki/github-discussions-templete#83] [Ideas]
![image](https://raw.githubusercontent.com/ishida-daiki/github-discussions-templete/main/images/ボタン_修正案.png)

添付した画像のように、丸みを帯びたボタンの提供をしてみてはいかがでしょうか？

https://www.figma.com/design/jV8iVvfK4tNP0sPv1LoxBC/Mobile%20Apps%20%E2%80%93%20Prototyping%20Kit%20(Community)?node-id=240-6286

Posted by: 石田 大貴

**Comment by KhangUlises4543D3V** [comment]
ああ、そうですね。
